### ISSUE TITLE
Esp32 WT32-ETH01

### ISSUE THREAD

**Issue by moshiko2312** [rednblkx/HomeKey-ESP32#60]
Hi how its possible to working with ESP32 WT32-ETH01
Its esp with LAN Cable.
I don't want working on WiFi
Thanks for all your job its the best way i see🙏

Just found this amazing item on AliExpress. Check it out! ₪37.61  67％ Off | WT32-ETH01 Embedded Serial Port Networking Ethernet Bluetooth-compatible WIFI Combo Gateway MCU ESP32 Wireless Module Board
https://a.aliexpress.com/_EvBJprt

**Comment by 011V32** [comment]
> Hi how its possible to working with ESP32 WT32-ETH01
> 
> Its esp with LAN Cable.
> 
> I don't want working on WiFi
> 
> Thanks for all your job its the best way i see🙏
> 
> 
> 
> Just found this amazing item on AliExpress. Check it out! ₪37.61  67％ Off | WT32-ETH01 Embedded Serial Port Networking Ethernet Bluetooth-compatible WIFI Combo Gateway MCU ESP32 Wireless Module Board
> 
> https://a.aliexpress.com/_EvBJprt

At the momement no LAN Support. But will come hopefully in near future. See other posts.

**Comment by moshiko2312** [comment]
Ok thanks
Its working perfectly

בתאריך יום ו׳, 31 במאי 2024 ב-14:56 מאת 011V32 ***@***.***>:

**Comment by moshiko2312** [comment]
I buy this one

https://shop.everythingsmart.io/products/est-poe-32-esp32-poe-development-board

**Comment by magic2910** [comment]
Is the board from aliexpress working?

**Comment by ChrisKoh83** [comment]
@moshiko2312 
To get this working I guess you had to build your own code to make it work?

because the GPIOs used by the Ethernet module!?

**Comment by DP365** [comment]
yep, correct. you need to use the second hardware SPI bus (called HSPI) because the first one (VSPI) is used by the ethernet module. See also my post here https://github.com/rednblkx/HomeKey-ESP32/issues/48#issue-2241602306

**Comment by klim123123** [comment]
Its just works for me. Nothing special, i used random GPIOs.

```
spi:
  clk_pin: GPIO14
  miso_pin: GPIO4
  mosi_pin: GPIO2

pn532_spi:
  id: nfc_spi_module
  cs_pin: 12
  update_interval: 100ms
```

**Comment by drstil** [comment]
> Its just works for me. Nothing special, i used random GPIOs.
> 
> ```
> spi:
>   clk_pin: GPIO14
>   miso_pin: GPIO4
>   mosi_pin: GPIO2
> 
> pn532_spi:
>   id: nfc_spi_module
>   cs_pin: 12
>   update_interval: 100ms
> ```

Thank you for this reply! How did you set those settings? `config.h`? How did you compile afterwards? Thank you very much!

**Comment by klim123123** [comment]
> @klim123123 Thank you for this reply! How did you set those settings? `config.h`? How did you compile afterwards? Thank you very much!

Sorry, i'm confused with other project of rednblkx.
I'm using rednblkx/HAP-ESPHome
Same functionality but for ESPHome. It's just works.
